src/main/java/org/apache/commons/io/FileUtils.java

```diff
@@ -3119,8 +3121,9 @@ private static int toMaxDepth(final boolean recursive) {
      * </p>
      *
      * @param file the File to touch.
-     * @throws IOException if an I/O problem occurs.
-     * @throws IOException if setting the last-modified time failed.
+     * @throws NullPointerException if the parameter is {@code null}
+     * @throws IOException          if an I/O problem occurs.
+     * @throws IOException          if setting the last-modified time failed.
```

Comment:
Listing each exception type once is enough IMO.

Reply:
I force pushed a new version merging the two descriptions for `IOException` in the same line. Does that work?